tsa-core/src/main/kotlin/org/usvm/machine/types/TvmDataCellTypesHandlers.kt

```diff
@@ -421,3 +431,19 @@ fun TvmStepScopeManager.storeSliceTlbLabelInBuilder(
         }
     }
 }
+
+fun TvmStepScopeManager.storeCellDataTlbLabelInBuilder(
+    oldBuilder: UConcreteHeapRef,
+    newBuilder: UConcreteHeapRef,
+    value: UExpr<TvmCellDataSort>,
+    sizeBits: UExpr<TvmSizeSort>,
+) = doWithCtx {
+    val cellRef = allocCellFromData(value, sizeBits)
+    val newSlice = calcOnState { memory.allocConcrete(TvmSliceType) }
+    doWithState {
+        memory.writeField(newSlice, TvmContext.sliceCellField, addressSort, cellRef, guard = trueExpr)
+        val refsInCell = memory.readField(cellRef, TvmContext.cellRefsLengthField, sizeSort)
+        memory.writeField(newSlice, TvmContext.sliceRefPosField, sizeSort, refsInCell, guard = trueExpr)
+    }
+    storeSliceTlbLabelInBuilder(oldBuilder, newBuilder, newSlice)
```

Comment:
This method should be used only if `value` or `sizeBits` is symbolic. Otherwise, you should use `addTlbConstantToBuilder` (in this case, there is no need to create `newSlice`)